node/Cargo.toml

```diff
@@ -4,6 +4,9 @@ version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
 
+[lib]
+crate-type = ["cdylib", "rlib"]
```

Comment:
Is `cdylib` needed because of the tests?

Reply:
It's needed for `wasm-pack build`. I see that tests run without it fine, so I'll remove it since we'll need to `build` `celestia`, not `node`.